.github/workflows/scripts/check-merge-freeze.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
+#
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+BLOCKERS=$( gh pr list --search "label:release:prepare" --json url --jq '.[].url' --repo "${REPO}" )
+if [ "${BLOCKERS}" != "" ]; then
+    echo "Merging in main is frozen while release PR is open: ${BLOCKERS}"
```

Comment:
Should this more explicitly talk about the label. Something like:
```suggestion
    echo "Merging in main is frozen since there are one or more PRs with the 'release:prepare' label: ${BLOCKERS}"
```